test/aes_256_encrypt.test.js

```diff
@@ -28,11 +29,30 @@ describe("AES256 Encrypt Test", () => {
 			out.push(Module.HEAPU32[out_ptr/Uint32Array.BYTES_PER_ELEMENT + i]);
 		}
 
-		let witness = await cir.calculateWitness({"ks":ks, "in":inp});
-		witness = witness.slice(1,5);
-		console.log("expected", out);
-		console.log("witness", witness);
+		var ks_buffer = [];
+		for(let i=0; i<ks.length; i++)
+        {
+            ks_buffer.push(...utils.intToLEBuffer(ks[i], 4));
+        }
+        var ks_bits = utils.buffer2bits(ks_buffer);
+
+		var inp_buffer = [];
+		for(let i=0; i<inp.length; i++)
+        {
+            inp_buffer.push(...utils.intToLEBuffer(inp[i], 4));
+        }
+        var inp_bits = utils.buffer2bits(inp_buffer);
+
+		var out_buffer = [];
+		for(let i=0; i<out.length; i++)
+        {
+            out_buffer.push(...utils.intToLEBuffer(out[i], 4));
+        }
+        var out_bits = utils.buffer2bits(out_buffer);
+
+		let witness = await cir.calculateWitness({"ks":ks_bits, "in":inp_bits});
+		witness = witness.slice(1,129);
 		
-		assert.ok(out.every((v, i)=> v == witness[i]));
+		assert.ok(out_bits.every((v, i)=> v == witness[i]));
```

Comment:
I think we should install eslint as part of our test pipeline to catch all the linting issue. @ultrainstinct30 @utsavjnn what do you think?

Reply:
Yep okay will add eslint.

Reply:
I would like if we make a separate PR for linting. This PR can only reflect why we made these changes and what changes we made. @ultrainstinct30 maybe add a lil description.